README.md

```diff
@@ -25,4 +25,21 @@ npm run dist:server
 ```
 
 ## Deployment
-TBD
+
+First build the app per instructions in the previous step, serve it locally, and make sure things look good.
+
+Then do either:
+
+```
+npm run dev-deploy
```

Comment:
Will do! Also noticed a typo in one of these commands. Also, will see if we know the bucket name for prod yet...